Microsoft.O365.Security.Native.libyara.NET.Core.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.O365.Security.Native.libyara.NET.Core</id>
+    <version>1.0.0</version>
```

Comment:
let's make this 4.0.1 to keep it in sync with the .NET Framework version

Reply:
good point